packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -125,7 +125,7 @@ <h2>Registrazione</h2>
       <div>
         In data <strong>{{submissionDate}}</strong> alle ore
         <strong>{{submissionTime}}</strong>, l’Infrastruttura ha trasmesso
-        all’erogatore la richiesta di fruizione.
+        {{#if producerDelegationId}} al delegante all’erogazione{{else}} all’erogatore{{/if}} la richiesta di fruizione.
```

Comment:
I like this new approach better than the approach you used for the [block below this](https://github.com/pagopa/interop-be-monorepo/pull/1243/files#diff-4a3319722f69e1c2ab048bde76e502ce96bb315fdafadd83a8029c9a06142464R97), where you rewrote the entire block in the if and else instead of just using the if-else in the sentence that should change. Could you align also that block and do the same?